Resources/Textures/_RMC14/Objects/dropship_attachments.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/83239d4edde7b1e8d59e950d07d47b84ee7508e4/icons/obj/structures/props/almayer_props64.dmi",
+  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/83239d4edde7b1e8d59e950d07d47b84ee7508e4/icons/obj/structures/props/almayer_props64.dmi and https://github.com/cmss13-devs/cmss13/blob/83239d4edde7b1e8d59e950d07d47b84ee7508e4/icons/obj/structures/props/almayer_props64.dmi",
```

Both links are the same
```suggestion
  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/83239d4edde7b1e8d59e950d07d47b84ee7508e4/icons/obj/structures/props/almayer_props64.dmi",
```